packages/lazy-images/.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Files not needed to be distributed in the package.
+.gitattributes export-ignore
+phpunit.xml    export-ignore
```

Comment:
Should it be `phpunit.xml.dist`?

Reply:
Yes you are right. Actually my mistake was due to copy the original .gitattributes file from the `connection` package which has this mistake and might need fixing too. I have now fixed that for this package.

Reply:
> Actually my mistake was due to copy the original .gitattributes file from the connection package which has this mistake and might need fixing too. 

Yes, that's a recent oversight:
https://github.com/Automattic/jetpack/pull/16810#issuecomment-673931819